gui-libs/wpebackend-fdo/files/wpebackend-fdo-1.14.2-c23-fix.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/ipc.h b/src/ipc.h
```

Comment:
This fix isn't related to C23. It's a GCC 15 because of transitive include changes in libstdc++, see https://gcc.gnu.org/gcc-15/porting_to.html#header-dep-changes.

If the change is a backport, please add a link to the upstream commit at the top of the patch, and ideally include the patch header (use the patch in full from https://github.com/whatever/foo/foo.patch).